test/position-managers/PositionManager.gas.t.sol

```diff
@@ -376,7 +376,13 @@ contract PosMGasTest is Test, PosmTestSetup, GasSnapshot {
 
     function test_gas_mint_native_excess() public {
         uint256 liquidityToAdd = 10_000 ether;
-        bytes memory calls = getMintEncoded(configNative, liquidityToAdd, address(this), ZERO_BYTES);
```

Comment:
if you wanted to keep using the liquidity operations helpers you could always have it check if one of the currencies is native and add a sweep automatically? like in `finalizeML` if its a mint/increase

Reply:
but dont feel strongly about it if you dnt want to

Reply:
eh its fine for now, can maybe add it later